<launch>

    <arg name="run_mloam_loop" default="true" />
    <arg name="result_save" default="true"/>
    <arg name="output_path" default="/home/jjiao/catkin_ws/src/localization/rpg_trajectory_evaluation/results/real_vehicle/oxford/20190118_800_2000/"/>

    <arg name="config_file" default="$(find mloam_loop)/config/config_loop_realvehicle.yaml" />
    <group if="$(arg run_mloam_loop)">
        <node pkg="mloam_loop" type="loop_closure_node" name="loop_closure_node"
            args="-config_file=$(arg config_file)
                  -result_save=$(arg result_save) 
                  -output_path=$(arg output_path)" output="screen">
        </node>
    </group>

 </launch>
